### PR TITLE
List drops from launchpad

### LINKED ISSUE #602: Upcoming & Past NFT Drops
Replace the current group of logos on the homepage with two sections of cards depicting upcoming and past drops. The past section should only show three but be expandable with a "View More" button. We probably don't need to worry about actual pagination yet. The cards should be left-aligned so that any remainders beyond the last multiple of three appears on the left rather than the center. All of the content should be fetched from a Launchpad API.

Going forward, the images are provided by creators via the builder (Twitter share image) but we may need to hardcode some for past drops or insert them into the Launchpad db. Rather than pull every event into this page automatically, they should require whitelisting via our admin dashboard. In addition to activating them for display on this page, we might also want to override the artist name and collection name shown under the image. Let's treat this section as a curated editorial rather than an automated feed.

For the upcoming drops, rather than showing a countdown or a specific time, I think we can just start with showing an abbreviated date (e.g. 6/12). The entire element (card + text) should be clickable and link to the URL of the drop's homepage.

### PR DESCRIPTION
**Pull request by cipherzzz**
Resolves #602 

Mocks
https://app.zeplin.io/project/59fa2311bac7acbc8d953da9/screen/60b5631f90f6103892ba6bac

Staging
https://staging-originprotocol-com.herokuapp.com/en/

### Description:
Adding upcoming/current/past drops to the `origin-website`. These are driven by the `launchpad` api from the `api/site-marketing` endpoint. Currently it's a curated list that is linked to a site and maintained inside the `launchpad.site_marketing` table. A UI is currently being built in the admin to manage this table, but it is being manually populated right now.

### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [ ] Update the README, if necessary
- [x] Run [translation commands](https://github.com/OriginProtocol/origin-website/tree/master/translations#extract-newedited-english-strings-for-translation) if any strings are changed

### CODE REVIEW

**Comment by joshfraser** [comment]
It looks like we need to populate some data before this can be merged.

File: logic/scripts/drops.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+from tools import db_utils
+from util import tasks
+from config import constants
+from types import SimpleNamespace
+from datetime import date, datetime, timezone
+from dateutil import parser
+
+
+
+headers = {
```
Comment (franckc) [inline logic/scripts/drops.py:19]: I'm curious. Are those headers actually required for the API call against the NFT server to succeed? Seems weird they would?

File: logic/scripts/drops.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+def get_drops(allPast):
+
+    headers = {
+    "Content-Type": "application/json",
+    }
+    url = "{0}/site-marketing".format(constants.LAUNCHPAD_API)
+
+    drops = []
+    try:
+        with closing(requests.get(url, headers=headers)) as resp:
```
Comment (franckc) [inline logic/scripts/drops.py:69]: The website can get under pretty heavy load. Especially the home page. Which will translate in high traffic on the NFT API server. I'd definitely recommend considering caching the data. And the data changes very infrequently (only when a new drop gets added), so it is a good candidate for applying caching. We have examples in this codebase for caching either in memory or in Redis. It is pretty straight forward.

**Review by franckc — APPROVED** [review]
Some comments inline otherwise LGTM. Caching would be good imho.

**Comment by cipherzzz** [comment]
@joshfraser - I'm working on a script, but need @micahalcorn to tell me what drops we want visible

File: logic/scripts/drops.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+def filter_past_drops(drops, allPast):
+    pastDrops = []
+    for drop in drops:
+        if not drop.endDate:
+            continue
+        if parse_date(drop.endDate) < datetime.now(timezone.utc):
+            pastDrops.append(drop) 
+    pastDrops = sort_drops(pastDrops)
+
+    print(allPast)
```
[the remark this answers is not in the document]
Comment (cipherzzz) [reply logic/scripts/drops.py:51]: yep thanks

File: logic/scripts/drops.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    print(len(pastDrops))
+
+    if allPast == 'true':
```
[the remark this answers is not in the document]
Comment (cipherzzz) [reply logic/scripts/drops.py:54]: did it a different way - thanks

File: logic/scripts/drops.py (hunk 1 above)
Comment (cipherzzz) [reply logic/scripts/drops.py:19]: I guess `application/json` is default so no... I removed them

File: logic/scripts/drops.py (hunk 2 above)
Comment (cipherzzz) [reply logic/scripts/drops.py]: I implemented an `in memory` cache. requested review to make sure it made sense to you. seems to be working for me. redis serialization was causing me problems...

**Review by franckc — APPROVED** [review]
In-memory cache implementation LGTM!
Thanks for adding that.

**Pull request merged**